### ISSUE TITLE
About boxes in video

### ISSUE THREAD

**Issue by czy-orange** [ylsung/VL_adapter#9]
Hi, @ylsung 
    Thanks for your sharing the code. I would like to know why boxes in video are all set as [0, 0, 0, 0] rather than [0, 0, 1, 1]. Any motivation for this setting? In my view, [0, 0, 1, 1] seems a more rational choice, as the whole frame is taken from video.

**Comment by ylsung** [comment]
The box information is used in original implementation of VLT5, where the inputs are objects extracted from Faster RCNN. Those box information indicates the position of object. However, in this repo, I used CLIP to extract the representation of an image/video, so there are no such box concept (because here are no objects) for the representation. We only use the position embedding at [here](https://github.com/ylsung/VL_adapter/blob/545fcbbdbbaec4c442de35567f6ae477ff4e8265/VL-T5/src/modeling_t5.py#L155) to indicate the position of patches/videos. I set all the boxes to zero just to disable the object position embeddings.
